PATHFIND/UpdateNavmeshBlockingObject.md

```diff
@@ -5,17 +5,17 @@ ns: PATHFIND
 
 ```c
 // 0x109E99373F290687 0x4E9776D0
-void UPDATE_NAVMESH_BLOCKING_OBJECT(int objectId, float posX, float posY, float posZ, float scaleX, float scaleY, float scaleZ, float heading, Any flags);
+void UPDATE_NAVMESH_BLOCKING_OBJECT(int objectId, float posX, float posY, float posZ, float scaleX, float scaleY, float scaleZ, float heading, int flags);
 ```
 
 ```
-| Flag Name      | Value | Description                                    |
-|----------------|-------|------------------------------------------------|
-| Default        | 0     |                                                |
-| Wander Path    | 1     | Blocking object will block wander paths        |
-| Shortest Path  | 2     | Blocking object will block (regular) shortest-paths |
-| Flee Path      | 4     | Blocking object will block flee paths          |
-| All Paths      | 7     | Blocking Object will block all paths           |
+enum BLOCKING_OBJECT_FLAGS {
+    BLOCKING_OBJECT_DEFAULT = 0,      // Default Flag
+    BLOCKING_OBJECT_WANDERPATH = 1,   // Blocking object will block wander paths
+    BLOCKING_OBJECT_SHORTESTPATH = 2, // Blocking object will block (regular) shortest-paths
+    BLOCKING_OBJECT_FLEEPATH = 4,     // Blocking object will block flee paths
+    BLOCKING_OBJECT_ALLPATHS = 7,     // Blocking object will block all paths
+}
```

Comment:
No need to add same enum, you can make a reference to `ADD_NAVMESH_BLOCKING_OBJECT` in my opinion.

Reply:
fair enough